.env.example

```
@@ -1,57 +1,77 @@
+DB_TYPE=postgres
+DB_HOST=localhost
+DB_PORT=5432
+DB_USERNAME=postgres
+DB_PASSWORD=your_password
+DB_NAME=beneficiary_backend
+
+# Keycloak Configuration
+# Keycloak server URL
+KEYCLOAK_URL=http://localhost:8080
+# Admin CLI client secret
+KEYCLOAK_ADMIN_CLI_CLIENT_SECRET=your_admin_cli_secret
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The KEYCLOAK_ADMIN_CLI_CLIENT_SECRET key should go before the KEYCLOAK_URL key

(UnorderedKey)